Add talk links and extend FAQ of code flow docs (#15707)
dotnet/arcade: Documentation/UnifiedBuild/Codeflow-PRs.md

- **Where can I find the official build of the VMR?**
  - The official build of the VMR is [dotnet-unified-build](https://dev.azure.com/dnceng/internal/_build?definitionId=1330).

- **What should I do in case of conflicts?**
  - Whenever there will be a conflict in a codeflow PR, Maestro bot will drop a comment in the PR guiding you through the process. If you're still unsure, you can always tag **@dotnet/product-construction** and we will help you resolve those.

- **My repo X depends on repo Y's packages. How will I get the new packages**?
  - If repo Y is part of the VMR, you will depend on the VMR instead of repo Y.
    The packages will be produced by the official VMR build and published to the `.NET 10 UB` channel.  
    The target feed for some packages might change from `dotnet-eng` to [`dotnet10-transport`](https://pkgs.dev.azure.com/dnceng/internal/_packaging/dotnet10-transport/nuget/v3/index.json).

    Furthermore, if you depend on multiple VMR repositories, you will get all the packages in a single backflow subscription (PR).

> - **What if I cannot accept a specific package update?**

Author: I hinted to use `darc update-subscription` but would not go further as that command has its own help and instructions so it's enough

>   - If your product repo is in the VMR but needs to stay on an older version of a certain package built in the VMR, the backflow PRs will keep updating it. You can specify this package in `Excluded assets` on the subscription for it to be ignored during backflow PRs.  
>     Alternatively, you can also pin the package in `Version.Details.xml` by adding the `Pinned="true"` attribute.

Author: ```suggestion
  - If your product repo is in the VMR but needs to stay on an older version of a certain package built in the VMR, the backflow PRs will keep updating it. You can specify this package in `Excluded assets` on the subscription settings (use `darc update-subscription`) for it to be ignored during backflow PRs.  
```

>     *Note: Excluding `Microsoft.DotNet.Arcade.Sdk` will also turn off udpates of the SDK, `eng/common` and other related toolset files.*

Author: Not really. Before, you would just not subscribe to arcade. But also I don't think it's a common knowledge that you can do that and so it doesn't hurt to mention
